assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -185,6 +198,7 @@ function Carousel() {
             return (
               <DraggablePage
                 key={index}
+                as="li"
```

Comment:
What's the benefit of this vs. just wrapping `DraggablePage` in a `li`?

Reply:
One wrapper less for every element. [as docs](https://styled-components.com/docs/api#as-polymorphic-prop)

Reply:
As long as it doesn't break functionality due to default styling and helps with semantics, this is a great, concise way to do it.